client/src/components/ChevronButton/ChevronButton.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+import "./ChevronButton.scss";
+
+function ChevronButton({ onClick, src, ariaLabel }) {
+  return (
+    <button
+      type="button"
+      aria-label={ariaLabel}

```

Comment:
If you have a decorative image, take advantage of the opportunity to specify the button's function in the alt attribute of the image contained in the button.